drivers/vulkan/rendering_device_vulkan.h

```diff
@@ -1068,6 +1069,8 @@ class RenderingDeviceVulkan : public RenderingDevice {
 
 	virtual FramebufferFormatID framebuffer_get_format(RID p_framebuffer);
 
+	virtual bool supports_vrs();
+
```

Comment:
we have Capabilities, which is kinda sloppy already, but I would add it there for now.

Reply:
I'll look into merging those, might indeed be nicer